### ISSUE TITLE
WEB SERVER ISSUE: 

### ISSUE THREAD

**Issue by B1sounours** [StrangeLoopGames/EcoIssues#4457]
**OS/Web browser**
Tested on chrome and firefox

**Version**
0.7.1.0 beta

**Expected behavior:**
Being able to open the "propose a new law" panel and able to create a new law

**Actual behavior:**
After the mayor was elected no one was able to open the "propose a new law" panel each time we get a red bar on top without any error message

https://i.imgur.com/dbSloXs.png

**Comment by stormoffires** [comment]
we get the same redbar when we try to run for mayor :(

Mediocre At Best world file
https://drive.google.com/open?id=1t1f2pFYn6IV3WMqcH4mdEftrcJOzTkWJ

**Comment by MarjieAVolk** [comment]
I looked at the save you sent, and (after removing the Citizenship law) I was able to both propose a law and run for office

**Comment by MarjieAVolk** [comment]
Is @B1sounours on the same server as @stormoffires ?

**Comment by B1sounours** [comment]
http://eco.aogservers.com/devdl/EcoDomeBetaServ.zip

**Comment by MarjieAVolk** [comment]
Okay I tested out this save too, and after deleting Citizenship I was able to propose laws and run for elections normally.  So I think this is the issue for both of you.

I just realized earlier today that because stats are currently disabled, the Citizenship law has stopped working, and doesn't let anyone do government actions.  So I think that is what you guys are seeing.  If you delete the Citizenship law on your server, does the problem go away?  (Admins can do this from the website).

We are working to get stats re-enabled asap, possibly they can be re-enabled by tomorrow.
